databroker/v1.py

```diff
@@ -1671,15 +1668,15 @@ def _get_mongo_database(config):
     uri = config.get('uri')
     database = config['database']
 
-    # If this statement is True than uri does not exist in the config.
+    # If this statement is True then uri does not exist in the config.
     # If the config has username and password, turn it into a uri.
     # This is only here for backward compatibility.
     if {'mongo_user', 'mongo_pwd', 'host', 'port'} <= set(config):
         uri = 'mongodb://{0}:{1}@{2}:{3}/'.format(
-                self.config['mongo_user'],
-                self.config['mongo_pwd'],
-                self.config['host'],
-                self.config['port'])
+            config['mongo_user'],
+            config['mongo_pwd'],
+            config['host'],
+            config['port'])
```

Comment:
Why not f-string here?

Reply:
I'll make a separate PR to make this the lightest touch possible to satisfy `flake8`.